tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -167,6 +170,32 @@ def should_exclude_files_type(file_key: str, exclude_file_types: List[str]) -> b
     return False
 
 
+def create_http_error_dict(
```

Comment:
Just remembered... This is only used for lambdas behind API Gateway. You can just let the errors raise.

Reply:
Concur, I feel this should be removed and the errors naturally allowed to buble up through the step function.